File: angular-movieFrontend/src/Data/Response/TvSeriesResponse.ts
import { GenreResponse } from "./GenreResponse";
import { MediaStatsResponse } from "./MediaStatsResponse";
import { ReviewResponse } from "./ReviewResponse";

export interface TvSeriesResponse {
    GenreResponse: any;
    mediaStats: MediaStatsResponse;
    seasons: number;
    episodes: number;
    network: string;
    status: string;
    id: string;
    title: string;
    description: string;
    
    genreResponse: GenreResponse;
    
    releaseDate: string;
    language: string;

    reviews: ReviewResponse[];
}

File: Application/Features/TvSeries MOZE EDYCJA MEDIASTATS/Common/TvSeriesResponse.cs
using Application.Features.Common.Interfaces;
using Application.Features.Genres.GetAll;
using Application.Features.Movies.Common;
using Application.Features.Reviews.Common;
using Domain.Enums;

namespace Application.Features.TvSeries.Common
{
    public record TvSeriesResponse(
        Guid id,
        string Title,
        string Description,
        GenreResponse Genre,
        DateTime ReleaseDate,
        string? Language,
        List<ReviewResponse>? Reviews,
        MediaStatsResponse MediaStats,
        int Seasons,
        int Episodes,
        string? Network,
        EStatus Status
        ): IResponse;
}
